tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -3107,13 +3107,12 @@ node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(60
       Type '(debuggerModel: DebuggerModel) => void' is not assignable to type '(model: T) => void'.
         Types of parameters 'debuggerModel' and 'model' are incompatible.
           Type 'T' is not assignable to type 'DebuggerModel'.
-node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(97,52): error TS2339: Property 'get' does not exist on type 'Multimap'.
-node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(120,52): error TS2339: Property 'valuesArray' does not exist on type 'Multimap'.
-node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(123,34): error TS2339: Property 'clear' does not exist on type 'Multimap'.
-node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(153,34): error TS2339: Property 'deleteAll' does not exist on type 'Multimap'.
+node_modules/chrome-devtools-frontend/front_end/bindings/BreakpointManager.js(97,56): error TS2345: Argument of type 'string' is not assignable to parameter of type 'K'.
```

Comment:
looks like we're moving on to "constructor functions are not really classes" bugs now, because those are uninstantiated type variables.